Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -8385,6 +8386,16 @@ void HTMLMediaElement::setOverridePreferredDynamicRangeMode(DynamicRangeMode mod
     player->setShouldDisableHDR(shouldDisableHDR());
 }
 
+void HTMLMediaElement::setPlatformDynamicRangeLimit(PlatformDynamicRangeLimit platformDynamicRangeLimit)
```

Comment:
I think this function should be renamed `setPlayerDynamicRangeLimit()`

Reply:
I'll go with `dynamicRangeLimitDidChange()`, it matches the one call from `RenderMedia::styleDidChange()`.